### DISCUSSION TITLE
Problem with scaling

### DISCUSSION THREAD

**Discussion by rubiin** [prasanthrangan/hyprdots#1254] [General]
![image](https://github.com/prasanthrangan/hyprdots/assets/8222059/45e7a793-0702-4365-90ad-c2502b4ea14d)
The rofi launcher seems big on my laptop
I have tried other iterations of of the settings below but these values look much good . The only issue is with rofi scaling. its way to big relative to the layput as you can see on the attached image

this is what i am using for scaling

```lua

xwayland {
    force_zero_scaling = true
    }

monitor=,highres,auto,1.3333

env = GDK_SCALE,1

```

Problem two is with not being able to size the cursor. Its also big and was trying to decrease it

**Comment by rubiin** [comment]
![image](https://github.com/prasanthrangan/hyprdots/assets/8222059/b453c4ef-163b-4b16-a8e0-05b9c7d0a2a6)

```
xwayland {
    force_zero_scaling = true
    }

monitor = eDP-1,highres,auto,1
env = GDK_SCALE,1

```

This fixes both the scaling for cursor and rofi but the swaybar is way small

**Reply by kRHYME7** [reply]
change waybar scaling 

Edit this file 

![image](https://github.com/prasanthrangan/hyprdots/assets/53417443/0a059ee9-ee59-4902-952e-9bbf02f8f6c8)

the second field is the waybar scaling and the 1st field is the state on which the control file is enable(1) or disabled (0)

**Reply by rubiin** [reply]
let me try will get back to you

**Reply by rubiin** [reply]
Do I have to edit the entry with 1 as first ?

**Reply by kRHYME7** [reply]
1 indicates the current waybar layout

**Reply by kRHYME7** [reply]
If I scale it to 60 
![image](https://github.com/prasanthrangan/hyprdots/assets/53417443/be7eede9-2dcb-4bf0-b525-4d66e0497ce0)

Scale to 28

![image](https://github.com/prasanthrangan/hyprdots/assets/53417443/122636fc-4e96-4a78-a8f7-2e3917d8aa4a)

**Reply by rubiin** [reply]
Tried setting to 60  no luck . 
![image](https://github.com/prasanthrangan/hyprdots/assets/8222059/6a229681-2110-4a2e-b186-531362633c5c)

**Reply by rubiin** [reply]
Also does waybar reloads on config change. I have been doing 
```
kill waybar
waybar & disown
```

**Reply by kRHYME7** [reply]
> kill waybar
waybar & disown


This is useless because the layout is not regenerated yet. 
Try to toggle it by using keybinds
![image](https://github.com/prasanthrangan/hyprdots/assets/53417443/6bdbf81c-bfdf-4030-af1c-c8df06559fef)


And  if you're on the main branch I think ``` Hyprdots waybar control ``` should work. It will watch any changes on the config.ctl and it will update waybar for you

**Reply by rubiin** [reply]
```
Hyprdots waybar contro
``` 
this works <3

**Reply by kRHYME7** [reply]
Maybe next week  this repo will be called 'Hyde' therefore the CLI will migrate from ``` Hyprdots ```  to ``` Hyde ``

**Reply by rubiin** [reply]
@kRHYME7 how did you open that popup in the screenshot? whats it called

**Reply by kRHYME7** [reply]
try to run  ``` Hyprdots show binds ``` or press the keyboard Icon on waybar

**Reply by rubiin** [reply]
Thanks man for the help . Finally I can use everything without any scaling issues so far
